src/index.html

```diff
@@ -6,9 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+
+    <header class="head">
+      <a href="#home" class="logo">
+        <img
+        class="icon"
+        src="images/logo.png"
+        alt="logo_img">
```

Comment:
```suggestion
        <img
          class="icon"
          src="images/logo.png"
          alt="logo_img"
        >
```
you did not fix the code style 🥲